examples/webgl_camera_cinematic.html

```diff
@@ -0,0 +1,246 @@
+					// gain: 2.0,
+					// bias: 0.5,
+					// fringe: 0.7,
+					//
+					// focalLength: 35,
+					// noise: true,
+					// pentagon: false,
+					//
+					// dithering: 0.0001
+
+				};
+
+				var matChanger = function( ) {
+
+					for (var e in effectController) {
+						if (e in camera.postprocessing.bokeh_uniforms)
+						camera.postprocessing.bokeh_uniforms[ e ].value = effectController[ e ];
+					}
+					camera.postprocessing.bokeh_uniforms[ 'znear' ].value = camera.near;
+					camera.postprocessing.bokeh_uniforms[ 'zfar' ].value = camera.far;
+					camera.setLens(effectController.focalLength, camera.frameHeight ,effectController.fstop, camera.coc);
+					effectController['focalDepth'] = camera.postprocessing.bokeh_uniforms["focalDepth"].value;
+				};
+
+
+				var gui = new dat.GUI();
+
+				gui.add( effectController, "focalLength", 1, 135, 0.01 ).onChange( matChanger );
+				gui.add( effectController, "fstop", 1.8, 22, 0.01 ).onChange( matChanger );
+				gui.add( effectController, "focalDepth", 0.1, 100, 0.001 ).onChange( matChanger );
+				gui.add( effectController, "showFocus", true ).onChange( matChanger );
+
+				matChanger();
+
+				window.addEventListener( 'resize', onWindowResize, false );
+
+			}
+
+
+		function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+			}
+
+			function onDocumentMouseMove( event ) {
+
+				event.preventDefault();
+
+				mouse.x = ( event.clientX / window.innerWidth ) * 2 - 1;
+				mouse.y = - ( event.clientY / window.innerHeight ) * 2 + 1;
+
+			}
+
+
+			function animate() {
+
+				requestAnimationFrame( animate, renderer.domElement );
+
+				render();
+				stats.update();
+
+			}
+
+
+			function render() {
+
+				theta += 0.1;
+
+				camera.position.x = radius * Math.sin( THREE.Math.degToRad( theta ) );
+				camera.position.y = radius * Math.sin( THREE.Math.degToRad( theta ) );
+				camera.position.z = radius * Math.cos( THREE.Math.degToRad( theta ) );
+				camera.lookAt( scene.position );
+
+				camera.updateMatrixWorld();
+
+				// find intersections
+
+				raycaster.setFromCamera( mouse, camera );
+
+				var intersects = raycaster.intersectObjects( scene.children );
+
+				if ( intersects.length > 0 ) {
+
+					var targetDistance = intersects[ 0 ].distance;
+					
+					//Using Cinematic camera focusAt method
+					camera.focusAt(targetDistance);
+
+					if ( INTERSECTED != intersects[ 0 ].object ) {
+
+						if ( INTERSECTED ) INTERSECTED.material.emissive.setHex( INTERSECTED.currentHex );
+
+						INTERSECTED = intersects[ 0 ].object;
+						INTERSECTED.currentHex = INTERSECTED.material.emissive.getHex();
+						INTERSECTED.material.emissive.setHex( 0xff0000 );
+					}
+
+				} else {
+
+					if ( INTERSECTED ) INTERSECTED.material.emissive.setHex( INTERSECTED.currentHex );
+
+					INTERSECTED = null;
+
+				}
+
+				renderer.render( scene, camera );
```

Comment:
@WestLangley yes, this line was there before I implemented CinematicCamera. Will change this too...